legal-api/src/legal_api/reports/business_document.py

```diff
@@ -518,7 +524,10 @@ def _format_address(address):
     def _set_meta_info(self, legal_entity: dict):
         legal_entity["environment"] = f"{self._get_environment()} BUSINESS #{self._legal_entity.identifier}".lstrip()
         legal_entity["meta_title"] = "Business Summary on {}".format(legal_entity["report_date_time"])
-        legal_entity["meta_subject"] = "{} ({})".format(self._legal_entity.legal_name, self._legal_entity.identifier)
+        if self._legal_entity.alternate_names and self._legal_entity.legal_name == None:
+            legal_entity["meta_subject"] = "{} ({})".format(self._legal_entity.alternate_names[0].name, self._legal_entity.identifier) 
+        else:
+            legal_entity["meta_subject"] = "{} ({})".format(self._legal_entity.legal_name, self._legal_entity.identifier)
 
```

Comment:
Kept getting an error here when the conditional was met, handles that case but needs improvement.